src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1585,6 +1585,13 @@ private function addMailerSection(ArrayNodeDefinition $rootNode)
                                 ->end()
                             ->end()
                         ->end()
+                        ->arrayNode('notification')
```

Comment:
this seems unrelated to this PR, but to the notifier component instead, isn't it ?

Reply:
It is related. It allows to change the default templates for the `NotificationEmail` class.